integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/passthru/transport/test/ESBJAVA3336HostHeaderValuePortCheckTestCase.java

```diff
@@ -91,11 +93,27 @@ public void stop() throws Exception {
      * @throws Exception
      */
     private void applyProperty(File srcFile, String key, String value) throws Exception {
-        File destinationFile = new File(srcFile.getName());
-        Properties properties = new Properties();
-        properties.load(new FileInputStream(srcFile));
-        properties.setProperty(key, value);
-        properties.store(new FileOutputStream(destinationFile), null);
-        serverConfigurationManager.applyConfigurationWithoutRestart(destinationFile);
+        FileInputStream fis = null;
+        FileOutputStream fos = null;
+        String outFileName = srcFile.getName();
+        try {
+            File destinationFile = new File(outFileName);
+            fis = new FileInputStream(srcFile);
+            fos = new FileOutputStream(destinationFile);
+            Properties properties = new Properties();
+            properties.load(fis);
+            properties.setProperty(key, value);
+            properties.store(fos, null);
+            serverConfigurationManager.applyConfigurationWithoutRestart(destinationFile);
+        } catch (Exception e) {
```

Comment:
what if applyConfiguration fails or any other error occurs? Do you still want to run the test case?

Reply:
Fixed. Please refer latest commit